pkgs/tools/networking/httping/default.nix

```diff
@@ -29,6 +29,6 @@ stdenv.mkDerivation rec {
     '';
     license = licenses.agpl3;
     maintainers = with maintainers; [ rickynils ];
-    platforms = platforms.linux;
+    platforms = [ "x86_64-linux" ] ++ platforms.darwin;
```

Comment:
Why restrict the package to x86_64-linux when it seems to work on all linux platforms, i.e. aarch64?

https://hydra.nixos.org/job/nixos/unstable-aarch64/nixpkgs.httping.aarch64-linux